src/Link/ApiLinkMacro.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Tomaj\NetteApi\Link;
+
+use Efabrica\Cms\Core\Macro\PageLink;
+use Latte\Compiler;
+use Latte\MacroNode;
+use Latte\Macros\MacroSet;
+use Latte\PhpWriter;
+use Tomaj\NetteApi\EndpointIdentifier;
+
+///**
+// * Usage in latte:
+// * {php $params = ['title' => 'My title', 'data-foo' => 'bar']}
+// * {apiLink $version, $package, $apiAction, $params}
+// */
+class ApiLinkMacro extends MacroSet
+{
+    public static function install(Compiler $compiler)
+    {
+        $macroSet = new static($compiler);
+        $macroSet->addMacro('apiLink',[ApiLinkMacro::class, 'start']);
+    }
+
+    public static function start(MacroNode $node, PhpWriter $writer)
+    {
+        $args = array_map('trim', explode(',', $node->args, 3));
+        $arguments = [
+            'version' => $args[0] ?? 'null',
+            'package' => $args[1] ?? 'null',
+            'action' => $args[2] ?? 'null',
+            'params' => $args[3] ?? '[]',
+        ];
+
+        return $writer->write('echo \Tomaj\NetteApi\Link\ApiLinkMacro::createLink($_presenter->context->getByType("' .
+            ApiLink::class . '"), ' .
+            $arguments['version']  . ', ' .
+            $arguments['package'] . ', ' .
+            $arguments['action'] . ', ' .
+            $arguments['params'] . ')');
+    }
+
+    /**
+     * @internal
+     */
+    public static function createLink(ApiLink $apiLink, $version, $package, $apiAction = '', $params = [])
+    {
+        $endpoint = new EndpointIdentifier('GET', $version, $package, $apiAction);
```

Comment:
toto GET tu je ok ze je napevno? nespravime aby sa to dalo nastavit?

Reply:
Mal som to najprv tak ze sa to dalo posielat, ale nemalo to ziadny vplyv na url co to vygenerovalo, tak som to vyhodil

Reply:
v htmlku mozes mat aj urlku na post (aj na hocico ine), napr. ked chces nejaky ajaxovy loader (load next) ktory je POSTovy

ten API link asi hlada endpointy ktore su zaregistrovane v decideri, takze ked tam das natvrdo GET, nikdy neurobis link na inu metodu

ci?

Reply:
tak mozes to asi pouzit aj pre post vo vo formulari napr ale url ti to vygeneruje rovnaku ci tam das post alebo get.

Reply:
ale nevygeneruje ju ak ju nemas zaregistrovanu